src/requirements/data/exams/ExamCredit.ts

```diff
@@ -389,12 +389,14 @@ function userDataToCourses(
   college: string,
   major: string,
   userData: ExamsTaken,
-  examType: 'AP' | 'IB'
+  examType: 'AP' | 'IB',
+  uniqueIdDecrementer: number
 ): CourseTaken[] {
   const userExams = userData[examType];
   const exams = examData[examType];
   const courses: CourseTaken[] = [];
-  userExams.forEach((userExam, examIndex) => {
+  let uniqueId = -2 - uniqueIdDecrementer;
```

Comment:
Why do we initialize `uniqueId` like this? A clarification comment would be helpful!